src/_lib/sass/src/_dropdowns.scss

```diff
@@ -217,6 +243,13 @@
                 top: 100%;
                 transform: rotateZ(180deg);
             }
+
+            // For nested dropdowns
+            .dropdown-submenu > .dropdown-menu {
```

Comment:
Not sure about the styles in here, `translateY(20%)` seems a little arbitrary, unless I'm missing something